LICENSE

```diff
@@ -1,7 +1,8 @@
 MIT License
 
-Copyright (c) 2018 Olivier Cailloux, Quentin Sauvage, Adèle Thorner, 
-  Laetitia Courgey, Vincent Navales
+Copyright (c) 2019 Théo Chennebault, Louis Ledoux, Emilie Chhean, Anael Dray,
```

Comment:
Please write: Copyright (c) 2019 Théo Chennebault, Louis etc. (2018) ; Olivier Cailloux, Latitia etc. (2019) (replace etc of course). Please order by alphabetical order of the last name inside each year (I know it was not, that’s a mistake).

Reply:
@oliviercailloux  Ok, will be done asap.